src/alpha/u16x2/wasm32.rs

```diff
@@ -0,0 +1,214 @@
+use std::arch::wasm32::*;
+
+use crate::pixels::U16x2;
+use crate::utils::foreach_with_pre_reading;
+use crate::{ImageView, ImageViewMut};
+
+use super::native;
+
+pub(crate) unsafe fn multiply_alpha(
+    src_image: &ImageView<U16x2>,
+    dst_image: &mut ImageViewMut<U16x2>,
+) {
+    let src_rows = src_image.iter_rows(0);
+    let dst_rows = dst_image.iter_rows_mut();
+
+    for (src_row, dst_row) in src_rows.zip(dst_rows) {
+        multiply_alpha_row(src_row, dst_row);
+    }
+}
+
+pub(crate) unsafe fn multiply_alpha_inplace(image: &mut ImageViewMut<U16x2>) {
+    for row in image.iter_rows_mut() {
+        multiply_alpha_row_inplace(row);
+    }
+}
+
+#[inline]
+pub(crate) unsafe fn multiply_alpha_row(src_row: &[U16x2], dst_row: &mut [U16x2]) {
```

Comment:
Adding the feature does not change the generated code for wasm32-wasi or wasm32-unknown-unknown.
From the link you posted: 
"Also note that if you call SIMD intrinsics but don’t enable SIMD via either of these mechanisms, you’ll still have SIMD generated in your program."

There could be a case where there is a different wasm32 target that does not support simd128, where this would be a useful annotation. Is it OK if I bring this up with some of my W3c colleagues?

Reply:
> "Also note that if you call SIMD intrinsics but don’t enable SIMD via either of these mechanisms, you’ll still have SIMD generated in your program."

Yes, that's all right. But with annotation all SIMD intrinsics will be inlined. My example on godbolt shows how this works.

Reply:
Done.